mash_scripts/katello/4.1.0/katello-4.1-el8.mash

```diff
@@ -0,0 +1,12 @@
+[katello-4.1-el8]
```

Comment:
Why 4.1.0 in the directory path? All others are x.y without .z.

Reply:
Look again, they are all x.y.z which matches what the code auto-generates.

Reply:
Hmm, right. Except 3.3. I do feel like we should move to x.y for simplicity and consistency with https://github.com/theforeman/tool_belt/tree/master/configs/katello though.